internal/pkg/errors/flink_errors.go

```diff
@@ -0,0 +1,58 @@
+package errors
```

Comment:
`flink` may be a better package for this struct, since it looks like that's the only place it's used?